split.Console/Interfaces/IDeckInterface.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+namespace split.Console.Interfaces
+{
+    /// <summary>
+    /// This interface is a rough draft. It will be implemented by an object
+    /// that encapsulates the actions of a Card deck. 
+    /// </summary>
+    public interface IDeckInterface
+    {
+        IDeckInterface shuffle();
```

Comment:
Not that I know of. The assumption is that the shuffling is shuffling. The only difference I am guessing is the number of cards you are shuffling. Some games, though none mentioned here, use fewer or more than 52 cards.